docs/content/getting-started/overview.mdx

```diff
@@ -0,0 +1,19 @@
+---
+title: "Overview | Dagster Docs"
+---
+
+# Overview
+
+This will be a brief overview of Dagster and include the following:
```

Comment:
Our current landing page (docs.dagster.io/getting-started) includes a lot of this already. I.e. it starts with answering "What's Dagster" and then points to a number of the resources listed below.

Are you imagining that this overview page would replace the current landing page or be in addition to it?

Reply:
Good question! Eventually I'd like this to be a replacement for the current state of the landing page. Ideally the landing page will become less text-heavy, more visual, and provide general guidance on what's in each category/how to use them.

Ex: Instead of a list of pages, some blocks that say things like "Guides - Help and step-by-step instructions for common use cases". I think this would help improve orienting new users with how/where to find what they need.

Josh, Dish, and I are (hopefully) gonna talk about a redesign in the next few weeks.

Reply:
Definitely open to change, but just want to call out that there's a lot about our current landing page that I really like: in particular that it starts with a short, plain-talk description of Dagster and follows with big links to the most important learning resources.

Reply:
Agreed! I like those aspects too. It's mostly the lists of links beneath that section I'm looking at possibly changing. As this is a big (and potentially very impactful) change, we'll definitely surface designs for feedback before deciding on anything.